wiring/src/spark_wiring_string.cpp

```diff
@@ -830,9 +830,3 @@ String String::format(const char* fmt, ...)
     }
     return result;
 }
-
-std::ostream& operator << ( std::ostream& os, const String& value ) {
```

Comment:
For reference: this was a source of a major binary size bloat due to the usage of iostreams, which was bringing std strings, locales, exceptions and system_error stuff into the binary no matter what.

Reply:
Is this a breaking change that we ought to announce? What would be typical usage? Something like `Serial << String("foo")` ? Once that's clear I can check with Studios if they use this pattern in any application.

Reply:
This is not a breaking change. DeviceOS does not support iostreams in any form. I have no explanation why that code was there if only for unit tests. For the example you listed to work `Serial` would have to be a subclass of `std::ostream`, which it isn't and none of the other wiring classes are.